Update learn guides with semver tool content (#4862)
ballerina-platform/ballerina-dev-website: learn/guides/publish-packages-to-ballerina-central.md

bal push
```

> ### Publish successive package versions 

Reviewer: Shall we say "Release a newer version of a package"?

Author: How about "Publish a newer version of a package", since this guide seems to have used the word "publish" instead of using "release" in almost everywhere? :) @praneesha WDYT?

Reviewer 2: Shall we say `Publish a new version of a package`?

bal push
```

### Publish successive package versions 

> If you require adding new features/improvements/fixes to a library package that you have already published to Ballerina central, 

Reviewer 2: ```suggestion
If you require adding new features/improvements/fixes to a library package, which you have already published to Ballerina central, 
```

> you are allowed to publish them under a new version, based on the [Semantic Versioning Specification](https://semver.org/).

Reviewer 2: ```suggestion
you are allowed to publish them under a new version based on the [Semantic Versioning specification](https://semver.org/).
```

> However, it's library developer's responsibility to be cautious when deciding new package versions

Reviewer 2: ```suggestion
>**Info:** It is the responsibility of the library developer to be cautious when deciding on the new package versions
```

Author: IMO we don't need to emphasise this paragraph as otherwise we'll have to do the same for the next paragraph as well. WDYT?

Reviewer 2: +1 Let's keep it as normal text.

> (especially when there are potential breaking/backward-incompatible API changes), as otherwise it may result in library

Reviewer 2: ```suggestion
(especially when there are potential breaking/backward-incompatible API changes), as otherwise, it may result in library
```

> versions that are compatible only by the version, but not by the implementation.

Reviewer 2: ```suggestion
versions that are compatible only by the version but not by the implementation.
```

As a precaution, you can use the Ballerina semver validator (an experimental CLI tool) to check if your new API changes

> conform to the version that you are trying to publish to the Ballerina central.

Reviewer 2: ```suggestion
conform to the version that you are trying to publish to Ballerina central.
```

> By default, running `bal semver` command on package root directory will compare the local version with the “closest

Reviewer 2: ```suggestion
By default, running the `bal semver` command on the root directory of the package will compare the local version with the `closest
```

> compatible” published version of the same package, available in the Ballerina Central. Executing `bal semver --help`

Reviewer 2: ```suggestion
compatible` published version of the same package available in Ballerina Central. Executing the `bal semver --help`
```

> command provides the help message that outlines all the available command options.

Reviewer 2: ```suggestion
command provides the help message, which outlines all the available command options.
```
